strawberryfields/backends/fockbackend/ops.py

```diff
@@ -251,27 +251,28 @@ def apply_gate_BLAS(mat, state, pure, modes, n, trunc):
             untranspose_list[transpose_list[i]] = i
 
         return np.transpose(ret, untranspose_list)
-    else:
-        if n == 1:
-            return np.dot(mat, np.dot(state, dagger(mat)))
 
-        # Transpose the state into the following form:
-        # |psi><psi||mode[0]>|mode[1]>...|mode[n]><mode[0]|<mode[1]|...<mode[n]|
-        transpose_list = [i for i in range(n*2) if not i//2 in modes]
-        transpose_list = transpose_list + [2*i for i in modes] + [2*i + 1 for i in modes]
-        view = np.transpose(state, transpose_list)
+    # otherwise, if state is mixed
```

Comment:
This might get confused with the below if statement. Maybe make this comment more direct?
```suggestion
    # otherwise, the state is mixed
```